tests/src/kakarot/accounts/test_contract_account.py

```diff
@@ -23,6 +45,10 @@ class TestInitialize:
         @SyscallHandler.patch("IKakarot.register_account", lambda addr, data: [])
         @SyscallHandler.patch("IKakarot.get_native_token", lambda addr, data: [0xDEAD])
         @SyscallHandler.patch("IERC20.approve", lambda addr, data: [1])
+        @SyscallHandler.patch(
+            "IKakarot.get_cairo1_helpers_class_hash",
```

Comment:
There is a `tests/utils/constants.py` file and I think that we could put therein a constant class hash to be used as placeholder everywhere for this helper class. And in the SyscallHandler class, we do use it as well to return (as discussed) the actual expected behavior (ie the value returned by the actual cairo1 class)